src/components/PDFView/ReactPDFPageRenderer.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React, {memo} from 'react';
+import {View} from 'react-native';
+import {Page} from 'react-pdf';
+import stylePropTypes from '@styles/stylePropTypes';
+import PDFViewConstants from './constants';
+
+const ReactPDFPageRenderer = memo(({index, data, style}) => {
```

Comment:
Just rename.
```suggestion
const PDFPageRenderer = memo(({index, data, style}) => {
```

Reply:
Thanks, indeed the destructuring with alias is a great idea. Regarding the naming convention, do you have in mind another candidate? I wanted to somehow mark that this is a web-only component, since for native we have a different library.

Reply:
I got your idea. Let's use the clearest name `WebPDFPageRenderer` 🙂

Reply:
Sounds good to me 😄

Reply:
Resolved in https://github.com/Expensify/App/pull/33083/commits/d9f90c12789e410017e04a75dae7db45267649af